docs/benchmarks.md

```diff
@@ -24,6 +24,45 @@ Benchmarks can be started from command line by the following command:
 make benchmark
 ```
 
+## Database benchmarks
+
+Database benchmarks can be run against local or remote database. The
+configuration is stored in
+[tests/benchmark.tom](https://github.com/RedHatInsights/ccx-notification-writer/blob/master/tests/benchmark.toml)
```

Comment:
wouldn't it be better to use
```markdown
[tests/benchmark.tom](tests/benchmark.toml)
```
?

Reply:
it should :thinking: why not?

Reply:
f.e https://github.com/RedHatInsights/ccx-notification-writer/blob/master/docs/architecture.md?plain=1#L10